experimental/InvariantTheory/secondary_invariants.jl

```diff
@@ -0,0 +1,509 @@
+export secondary_invariants, irreducible_secondary_invariants
+
+################################################################################
+#
+#  Helpers
+#
+################################################################################
+
+# Add a base element to the cache.
+function add_base_element!(C::PowerProductCache{S, T}, f::T, remember_exponents::Bool = true) where {S, T}
+  @assert !iszero(f)
+
+  push!(C.base, f)
+  if remember_exponents
+    for (g, exps) in C.exponent_vectors
+      push!(exps, 0)
+    end
```

Comment:
Is `remember_exponents` always `true`? Seems to not be covered by the tests.

Reply:
See also below.

Reply:
I don't know the details, but if you don't use it here, is there any possible application where this would be useful?